sass/components/_content_project.scss

```diff
@@ -10,3 +10,6 @@
     margin-left: 0;
   }
 }
+div#project-details div.ecl-col-12.ecl-col-md-6 div + div.ecl-editor {
+  margin-top: $ecl-spacing-2-xs;
+}
```

Comment:
Instead of doing this can't we add margin bottom classes the each of this field templates?

- featured media
- body

![image](https://user-images.githubusercontent.com/153362/94273430-b489a500-ff44-11ea-983f-1f9dd54c75e8.png)